docs/api/errors.rst

```diff
@@ -52,8 +52,8 @@ These classes are exposed both by this module and the root `psycopg` module.
 
         Most likely it will be in `~psycopg.pq.ConnStatus.BAD` state;
         however it might be useful to verify precisely what went wrong, for
-        instance checking the `~psycopg.pq.PGconn.needs_password` and
-        `~psycopg.pq.PGconn.used_password`.
+        instance checking the :attr:`~psycopg.pq.PGconn.needs_password` and
+        :attr:`~psycopg.pq.PGconn.used_password` attributes.
```

Comment:
The `:attr:` shouldn't be necessary, because the default interpreted text role is `:obj:`.